.circleci/config.yml

```diff
@@ -384,6 +384,43 @@ jobs:
           path: build/test-results/test/TEST-org.mifos.integrationtest.TestRunner.xml
       - store_artifacts:
           path: build/test-results
+  test-chart-ci:
```

Comment:
lets remove the other two sections, common/gov/ams and keep only ci

Reply:
Done!